sass/components/button/_button.scss

```diff
@@ -20,6 +20,7 @@ $default-component-button-properties:(
   textAlign: center,
   disabledColor: #dddddd,
   disabledTextColor: getThemeProperty(textColorLight),
+  lineHeight: 1,
```

Comment:
Maybe the default should be "inherit". This doesn't cause issues with previous sites (I tested with fairview evergreen).

Reply:
Yes, I think this makes sense